lib/honeydew/queue.ex

```diff
@@ -117,15 +117,11 @@ defmodule Honeydew.Queue do
 
   def handle_cast(:resume, %State{suspended: false} = state), do: {:noreply, state}
   def handle_cast(:resume, state) do
-    # resume(state)
```

Comment:
would you just mind restoring the commented out function calls?

i left them in as a reminder that it might be necessary for queue implementations that push batches rather than pull to notify their upstream pusher to stop sending jobs

Reply:
Absolutely. I'll push a change right now.